content/docs/nav.yml

```diff
@@ -1,89 +1,89 @@
-- title: Installation
+- title: Instalacija
   items:
     - id: getting-started
-      title: Getting Started
+      title: Prvi koraci
     - id: add-react-to-a-website
-      title: Add React to a Website
+      title: Dodavanje React-a
     - id: create-a-new-react-app
-      title: Create a New React App
+      title: Nova React aplikacija
     - id: cdn-links
-      title: CDN Links
+      title: CDN linkovi
     - id: release-channels
       title: Release Channels
-- title: Main Concepts
+- title: Glavni koncepti
   isOrdered: true
   items:
     - id: hello-world
-      title: Hello World
+      title: Zdravo, svete
     - id: introducing-jsx
-      title: Introducing JSX
+      title: Upoznavanje JSX
     - id: rendering-elements
-      title: Rendering Elements
+      title: Prikazivanje elemenata
     - id: components-and-props
-      title: Components and Props
+      title: Komponente i svojstva
     - id: state-and-lifecycle
-      title: State and Lifecycle
+      title: Stanje i životni vek
     - id: handling-events
-      title: Handling Events
+      title: Rukovanje događajima
     - id: conditional-rendering
-      title: Conditional Rendering
+      title: Uslovno prikazivanje
     - id: lists-and-keys
-      title: Lists and Keys
+      title: Liste i ključevi
     - id: forms
-      title: Forms
+      title: Forme
     - id: lifting-state-up
-      title: Lifting State Up
+      title: Uzdizanje stanja naviše
```

Comment:
It's not `uzdizanje` but `prenosenje`. 
Also `stanje` is a literal translation of `state` but here `state` means something else. 
I think we should first define how we handle those keywords.

Reply:
I agree with this, we should decide now at the beginning whether or not we're translating keywords like _state_, _props_, _refs_, _reconciliation_, etc.

Reply:
@Milos5611 @vaske 
I've looked around some other translations of the React website, it seems as though everybody has been translating **_state_** as variations of _lokalni podaci_, and **_props_** as _ulazni podaci_.
For stuff like _refs_, _reconciliation_ and others, it's about 50/50. The French translation has them translated, whereas the German translation just left them as they were in English.

How do you guys propose we proceed? We need to figure this out quickly if we want to continue with the translations.

Reply:
I don't know. For me `state`, `props`, `refs`, etc...are like `var`, `const`...reserved words.

Reply:
I pretty much agree with @Milos5611 in this situation I would rather keep it as it is, because in code you have to use it like that.

Reply:
Okay, we'll do it like that, then.
We'll treat them as reserved words, but where applicable explain in depth what they are (in the tutorial sections).
I'll refactor this code and commit it, so that you can merge it into the repo.